### PR TITLE
[MIG] Port account_journal_always_check_date to v8 and new API

### PR DESCRIPTION
**Pull request by alexis-via**
and also fix a typo in README.md

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1: 

Thanks for the migration.

File: account_journal_always_check_date/__openerp__.py [hunk 1]
```diff
@@ -49,7 +49,6 @@
     'website': 'http://www.akretion.com',
     'depends': ['account'],
     'data': [],
-    'images': ['images/always_check_date_constraint.jpg'],
```
Comment (rvalyi) [inline account_journal_always_check_date/__openerp__.py:52]: you are trashing the image entry but you keep the file and references it nowhere. Is that wanted? May be you could switch for README.md now that it is supported and use the image from it?

**Comment by alexis-via** [comment]
I trashed the 'image' key in **openerp**.py because it seems to be deprecated in v8. I moved the screenshots to static/description because it seems that it's where they are supposed to be. I may write a README.md one day when I have time to take advantage of these screenshot, but it will not be in this PR. This PR aims at getting this module out of the **unported** directory ASAP.

**Comment by rvalyi** [comment]
:+1:  well at least I tried ;-) OK for migrating first and putting the lipstick second.

**Pull request merged**